lib/generators/statesman/templates/create_migration.rb.erb

```diff
@@ -5,9 +5,11 @@ class Create<%= migration_class_name %> < ActiveRecord::Migration
       t.text :metadata<%= ", default: \"{}\"" unless mysql? %>
       t.integer :sort_key, null: false
       t.integer :<%= parent_id %>, null: false
+      t.string :most_recent, default: 't', limit: 1, null: false
       t.timestamps null: false
     end
 
-    add_index :<%= table_name %>, [:<%= parent_id %>, :sort_key], unique: true, name: "<%= index_name :parent_sort  %>"
+    add_index :<%= table_name %>, [:<%= parent_id %>, :sort_key], unique: true, name: "<%= index_name :parent_sort %>"
+    add_index :<%= table_name %>, [:<%= parent_id %>, :most_recent], unique: true, where: "most_recent = true", name: "<%= index_name :parent_most_recent %>"
```

Comment:
Using `where: "most_recent"` should avoid the boolean literal compatibility issue here.